lib/Slackbot_worker.js

```diff
@@ -131,10 +131,14 @@ module.exports = function(botkit, config) {
 
             bot.identity = res.self;
             bot.team_info = res.team;
+            bot.user_info = {};
+            res.users.forEach(userRecord => {
```

Comment:
What if `res.users` is undefined or not an array? Can you write this code defensively?